packages/async/src/index.story.test.ts

```diff
@@ -1,5 +1,4 @@
-import { test } from 'uvu'
-import * as assert from 'uvu/assert'
+import { describe, it, expect } from 'vitest'
```

Comment:
btw, you can set up a `global: true` in vitest.config and this will allow you to remove all imports from 'vitest'

https://vitest.dev/config/#globals

Reply:
imho direct import is more obvious.